website/index.html

```diff
@@ -889,135 +846,55 @@
     <!-- Gold Sponsors -->
     <div class="row gold d-flex justify-content-center" style="padding-top: 2%">
       <div class="col-sm-6">
-      <!-- Geico -->
+        <!-- Geico -->
         <a href="https://www.geico.com/" target="_blank">
           <img src="assets/images/sponsors/Geico-day.svg" alt="Geico" class="img-fluid Geico-night">
         </a>
       </div>
-      <div class="col-sm-6">
-        <!-- GCP -->
-        <a href="https://cloud.google.com/" target="_blank">
-          <img src="assets/images/sponsors/gcp_F19.svg" alt="GCP" class="img-fluid GCP-night">
-        </a>
-      </div>
+
       <div class="col-sm-6">
         <!-- Textron -->
         <a href="https://cloud.google.com/" target="_blank">
-          <img src="assets/images/sponsors/textron_day.svg" alt="textron" class="img-fluid textron-night">
+          <img src="assets/images/sponsors/textron_day.svg" alt="textron" class="img-fluid textron-night" style="padding-top: 12%">
```

Comment:
I really dislike the way all of these sponsor images are being rendered. For now I think we should commit, but in the future I think I'll try to rewrite the layout for these sponsor images so that there aren't so many weird formatting problems.

Reply:
sounds like a good task!